templates/opennextjs/nextal-next-typescript-tailwind/info.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "Nextal - (NextJS, Typescript, Tailwind)",
+  "message": "Starter template for Next with TypeScript. Supports Tailwind with CSS-Modules. Vitest and @react/testing-library configured and ready to go. Also ESLint, Prettier, Husky, Commit-lint and Atomic Design for components.",
```

Let's keep this under an original tweet. 😃 

```suggestion
  "message": "Starter template for Next with TypeScript. Supports Tailwind with CSS-Modules.",
```